package.json

```diff
@@ -52,24 +51,14 @@
     "@types/jest": "^29.5.6",
     "@types/node": "^20.8.0",
     "@types/react-native": "^0.65.0",
-    "@typescript-eslint/eslint-plugin": "^6.20.0",
-    "@typescript-eslint/parser": "^6.8.0",
     "acorn-jsx": "^5.3.2",
     "babel-loader": "^9.1.3",
     "babel-plugin-react-native-web": "^0.19.13",
-    "eslint": "^8.53.0",
-    "eslint-plugin-import": "^2.27.5",
-    "eslint-plugin-react": "^7.33.2",
-    "eslint-plugin-react-hooks": "^4.6.0",
-    "husky": "^9.1.7",
```

Comment:
Sorry, my merge commit removed husky. Biome has an official document for Git Hooks - could you restore husky + lint-staged, or move to another library if it's better?

https://biomejs.dev/recipes/git-hooks/#husky